mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule MixTestInteractive.MixProject do
   use Mix.Project
 
-  @version "2.0.3"
+  @version "2.0.4-dev.1"
```

Comment:
Please don't adjust the version number here. I'll do that as part of the release process.